macros/console/region-and-az.mdx

```diff
@@ -2,8 +2,8 @@
 macro: region-and-az
 ---
 
-A Region is as a geographical area such as France (Paris: `fr-par`) or the Netherlands (Amsterdam: `nl-ams`) in which Scaleway products and resources are located. It can contain multiple Availability Zones.
+A Region is a geographical area such as France (Paris: `fr-par`) or the Netherlands (Amsterdam: `nl-ams`) in which Scaleway products and resources are located. It can contain multiple Availability Zones.
```

Comment:
```suggestion
A region is a geographical area such as France (Paris: `fr-par`) or the Netherlands (Amsterdam: `nl-ams`) in which Scaleway products and resources are located. It can contain multiple Availability Zones.
```